src/main/cpp/RobotContainer.cpp

```diff
@@ -195,10 +220,15 @@ bool RobotContainer::InitCommands() {
     //m_autonomousCommand = std::make_shared<AutoSwerveCommand>(swerve_drive_.get(), frc::Pose2d());
     m_autonomousCommand_ = nullptr;
 
+    // swerve commands
     swerve_teleop_command_ = std::make_shared<TeleOpSwerveCommand>(swerve_drive_, gamepad1_);
 
-    manual_arm_command_ =std::make_shared<ManualArmCommand>(arm_, gamepad2_);
-    arm_->SetDefaultCommand(*manual_arm_command_);
+    // arm commands
+    extendArmCommand = std::make_shared<IncrementArmExtendCommand>(arm_, 3); 
+    retractArmCommand = std::make_shared<IncrementArmExtendCommand>(arm_, -3);
 
+    raiseArmCommand = std::make_shared<IncrementArmPresetPositionCommand>(arm_, 3);
+    lowerArmCommand = std::make_shared<IncrementArmPresetPositionCommand>(arm_, -3);
```

Comment:
In the interest of making progress, let's add these to the aforementioned ticket about making these arm numbers into parameters

Reply:
#45